src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -57,10 +59,15 @@ object Run  {
   }
 }
 
-case class Run(name: String, pipeline: Pipeline, tag: String) {
+case class Run(jesId: String, pipeline: Pipeline, tag: String) {
+
+  val dataAccess = DataAccess()
```

Comment:
`withDataAccess` is only (and I believe probably should only) be called from tests; we might want to rework that so it doesn't live in production code where it encourages creating multiple copies.